src/file.ts

```diff
@@ -12,22 +12,6 @@ import { _chown, Stats } from './stats.js';
 	@todo Remove this if TS adds them to lib declarations
 */
 declare global {
-	interface ArrayBuffer {
-		readonly resizable: boolean;
-
-		readonly maxByteLength?: number;
-
-		resize(newLength: number): void;
-	}
-
-	interface SharedArrayBuffer {
-		readonly resizable: boolean;
-
-		readonly maxByteLength?: number;
-
-		resize(newLength: number): void;
-	}
-
 	interface ArrayBufferConstructor {
 		new (byteLength: number, options: { maxByteLength?: number }): ArrayBuffer;
```

Comment:
This should be able to be removed as well.

Reply:
Fixed in [`9461e61` (#149)](https://github.com/zen-fs/core/pull/149/commits/9461e61272f476c6426007aa3d1efccf46884c15)